### PR TITLE
Remove duplicate blocks from changes_list before import

### PR DESCRIPTION
**Pull request by pasqu4le**
## Motivation

Solves error arising (mostly started by block_uncle fetcher) when the same block is present twice in the same changes_list

## Checklist for your PR

  - [x] I added an entry to `CHANGELOG.md` with this PR
  - [x] If I added new functionality, I added tests covering it.
  - [x] If I fixed a bug, I added a regression test to prevent the bug from silently reappearing again.
  - [x] I checked whether I should update the docs and did so if necessary
  - [x] If I added/changed/removed ENV var, I should update the list of env vars in https://github.com/poanetwork/blockscout/blob/master/docs/env-variables.md to reflect changes in the table here https://poanetwork.github.io/blockscout/#/env-variables?id=blockscout-env-variables. I've set `master` in the `Version` column.
  - [x] If I add new indices into DB, I checked, that they don't redundant with PGHero or other tools

### CODE REVIEW

**Comment by ayrat555** [comment]
are you sure this is only happening with uncles?

**Comment by pasqu4le** [comment]
@ayrat555 it looks like it, uncles is the only one where this is happening (from the logs I have). The fix modifies the runner anyway so if this is happening with other block-inserting fetchers as well if will be de-duplicated anyway.

I am looking into why this fetcher is the only one to be affected, as like the other should have the filtering before fetching and should not end up with duplicates during importing

File: apps/explorer/lib/explorer/chain/import/runner/blocks.ex [hunk 1]
```diff
@@ -224,7 +224,10 @@ defmodule Explorer.Chain.Import.Runner.Blocks do
     on_conflict = Map.get_lazy(options, :on_conflict, &default_on_conflict/0)
 
     # Enforce Block ShareLocks order (see docs: sharelocks.md)
-    ordered_changes_list = Enum.sort_by(changes_list, & &1.hash)
+    ordered_changes_list =
+      changes_list
+      |> Enum.sort_by(& &1.hash)
```
Comment (ayrat555) [inline apps/explorer/lib/explorer/chain/import/runner/blocks.ex]: `Enum.uniq_by`?

Comment (pasqu4le) [reply apps/explorer/lib/explorer/chain/import/runner/blocks.ex]: That would work for the duplicates, but they also need to be sorted

**Pull request merged**